### DISCUSSION TITLE
Meeting list is empty

### DISCUSSION THREAD

**Discussion by COMOdistrict12** [code4recovery/12-step-meeting-list#766] [Q&A] [answered]
### Contact Details

webmaster@district12.eamo.org

### Website With Issue

https://district12.eamo.org/meetings/

### What happened?

We use hostgator as our host.  They recently updated us to PHP 7.4 and our TSML UI has stopped working.  All of the meeting data is still in the database, but the WP Plugin isn't displaying the meetings on our website.  
We tried downgrading to PHP 7.3 but didn't seem to help.  We made sure WP, pro theme, and all plugins are up to date.  Any suggestions to try?  

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

**Comment by joshreisner** [comment]
@COMOdistrict12 hi! to clarify, hopefully, TSML UI is something else -- you're using the "Classic Interface" on that page

try going to a meeting listing and hitting "update" (even if there are no changes) -- i suspect the local TSML cache may be out of date

**Comment by kiyote33** [comment]
Hi @COMOdistrict12,
Thanks for reporting this issue. Can you try resaving the site Settings/permalinks? That has helped others in the past.  Let us know if works.

**Comment by kiyote33** [comment]
@COMOdistrict12 Please note: For our work flow, we ask that user issues be opened inside of Discussions first. The maintainers will then migrate the details to an Issue when appropriate. I have converted this issue to a discussion topic for posterity, Thanks!

In the meantime, can we get more information on the problem you are reporting.  I see the link you provided ([https://district12.eamo.org/meetings/]) is listing meetings with the Legacy UI. What exactly is the problem you are reporting now? Please try and provide enough information that we might replicate your situation, if that's possible.

**Comment by COMOdistrict12** [comment]
The issue started when our Host (Host Gator) upgraded our version of PHP.
Meetings from the database were not being shown on the website.  You could
search but there weren't any meetings to show.   We upgraded and downgraded
everything that would let us: WP, PHP, plugins, nothing was working.

Our meeting list is up and working again, the crisis level has been reduced
down to amber.

This is what I received back from the alternate webmaster who tried your
suggestions:
"I chose a couple of Friday meetings to update and after a couple burps the
meeting list appeared. It took selecting and reselecting different days for
the page to auto-refresh on selection.  Then, I resaved
Settings/permalinks."

**Reply by kiyote33** [reply]
Hi @COMOdistrict12, if I understand correctly, your alternate webmaster reports the District 12 website is now displaying correctly after resaving the permalinks. That lines up with what we are seeing. Unless you have something else, I'll mark this discussion as resolved?

**Reply by thessnake03** [reply]
this issue is unresloved. see below

**Comment by thessnake03** [comment]
This issue is unresolved.  

District 12 is a subdomain of the area website, eamo.org.  Now district 12 meeting guide info (or the most recently updated) is the only meeting info displayed across any of the subdomains.

https://district12.eamo.org/meetings/
https://eamo.org/meetings/
https://district17.eamo.org/meetings/?tsml-day=any

**Comment by kiyote33** [comment]
@thessnake03 It appears the same solution applied by district 12 is required on all the other sub-domain sites. A site administrator needs to simply press the Save Changes button on the Settings/Permalinks page for each of the sub-domain sites. Please advise if this does not resolve the situation for you.

**Reply by thessnake03** [reply]
saving changes in settings/permalinks in those sites indiviually did not resolve the issue

**Reply by thessnake03** [reply]
I'll save changes on subdomains even without meeting info, not all our subdomains use the plugin but it is active across our install

**Reply by kiyote33** [reply]
One of our mainiainers suggests that it may be a caching issue. You can force a refresh of the cache by doing a SAVE of any meeting.

**Reply by thessnake03** [reply]
save changes in settings/permalinks across all 15 subdomains did not solve the issue

**Reply by kiyote33** [reply]
We have a bugfix ([#764 Bugfix/Append blog ID to meeting cache file name](https://github.com/code4recovery/12-step-meeting-list/pull/764)) which addresses the caching problem on multisite wordpress instances. We may need to fast track the update to help resolve this.

**Reply by thessnake03** [reply]
making an update to a meeting, or creating a new one, pushes that subdomains meeting info out across all of the subdomains, district 12 is showing for district 12, eamo, and district 17

**Reply by thessnake03** [reply]
a fix for multisite caching would be appreciated :D

**Reply by kiyote33** [reply]
Is rolling back to 3.13 and option until we can get a 3.14.1 patch released?

**Reply by kiyote33** [reply]
Or do you have a comfort level with implementing a code change (via FTP)? The maintainer reports the patch should be released later today.

**Reply by thessnake03** [reply]
personally, i think id be more comfortable rolling back to 3.13.  im not an it professional by any means, just a tech volunteer that has picked up a bit of wordpress.

**Reply by kiyote33** [reply]
👍

**Reply by thessnake03** [reply]
Can you walk me through rolling back?  That was one of the first options I looked into when I saw there was a release a few days ago, but there's not rollback button lol

**Reply by thessnake03** [reply]
Where can I find the 3.13 files?

**Reply by kiyote33** [reply]
Sorry @thessnake03, I got busy testing the 3.14.1 patch in a multisite environment and didn't see your request.  When wanting to rollback a plugin version I use the WP Rollback plugin which simplifies the whole process for me. Once installed it provides a Rollback link on each plugin in the installed plugin folder. It will handle getting a list of previous versions for you. Let me know if there's anything else I can do for you...

**Reply by thessnake03** [reply]
Rolling back to 3.13 took care of the issue (and thank you for the heads up on the rollback plugin).  I look forward to the full fix in a future release.

Thanks for all your help and guidance :D

**Reply by brianw-area24** [reply]
Version 3.14.1, intended to fix this issue, was released a few minutes ago. When you get a chance please upgrade to this version and let us know if it works. You can always roll back to 3.13 again if you need to, although I don't think that will be necessary. This release fixes a bug I introduced in 3.14 because I hadn't considered multisite installations.

**Reply by thessnake03** [reply]
Interestingly enough 3.14.1 works for the subdomains but not the main site

**Reply by kiyote33** [reply]
Hi @thessnake03 Probably because of the new unique cache file name, I had to reload my sub-domain (district) data sources in the parent multisite after the upgrade.  Seemed to work correctly for me.  Let us know if it's still an issue for you after doing a refresh of your district data sources in the main site.

**Comment by kiyote33** [comment, marked as answer]
A review of the [eamo.org](https://eamo.org) website shows our plugin working correctly. The page source shows it updated to [3.14.1]. Unless we hear otherwise, we can close this discussion as resolved by the latest patch.
